scripts/setup.sh

```diff
@@ -195,7 +195,7 @@ elif [ "$VAGRANT" = true ]; then
   # XVFB is required for headless testing in Vagrant
   sudo apt-get install xvfb chromium-browser
   export CHROME_BIN="/usr/bin/chromium-browser"
-  # Used in frontend and e2e tests. Only gets set if using Vagrant VM.
+  # Used in frontend and e2e tests. Only gets set if using Vagrant VM.   
```

Comment:
Why do you need this here though? Can't it go in run_performance_tests.sh? Confusing to set the same env var in two different places.